docs/Lua API.rst

```diff
@@ -1073,6 +1088,15 @@ Announcements
   Uses the type to look up options from announcements.txt, and calls the above
   operations accordingly. The units are used to call ``addCombatReportAuto``.
 
+* ``dfhack.gui.autoDFAnnouncement(report,text[,log_failures])``
+  ``dfhack.gui.autoDFAnnouncement(type,pos,text,color[,is_bright,unit1,unit2,is_sparring,log_failures])``
+
+  Takes a ``df.report_init`` (see: https://github.com/DFHack/df-structures/blob/master/df.announcements.xml#L451)
```

Comment:
Sure.

Reply:
I also want to point out that, either way, this link is not stable because right line numbers on `master` can change. And if we use a permalink to a specific commit, it could also become out of date by linking to an old/inaccurate definition. Not too sure what the best solution is, since we don't have docs for DF types, so I guess this is ok for now. At least the filename should be stable for the time being.